unicorn/src/service/unicorn.cpp

```diff
@@ -67,7 +68,12 @@ class unicorn_slot_t :
             try {
                 result.write(future.get());
             } catch (const std::system_error& e) {
-                result.abort(e.code(), error::to_string(e));
+                COCAINE_LOG_WARNING(service->log, "could not assign to deferred - {}", error::to_string(e));
```

Comment:
In the log it will be strange to see just `could not assign ...` without context.